src/pages/monetization/app-review-process/index.mdx

```diff
@@ -26,37 +26,74 @@ During this process, we will make sure that:
 - The app matches the LiveChat's look and feel. For that reason, we highly recommend following our Design System.
 - The app has all the necessary Marketplace listing details: app icon, images, description, key features and benefits, a tutorial, and privacy policy.
 
-# Best practices
+## App quality requirements
+
+There are several key requirements that determine whether your application will successfully pass the app review. These guidelines ensure that your app meets the necessary standards for functionality, user experience, and overall quality. All developers must adhere to these requirements in order for their apps to be able to publish their apps on the Marketplace.
+
+### Design System
```

Comment:
We require apps to be consistent with the Text look & feel. All new apps submitted to Text Marketplace after Jan 1, 2025 should use our Design System.